src/main/java/de/teamlapen/vampirism/player/vampire/VampirePlayer.java

```diff
@@ -601,9 +622,6 @@ public int onBite(IVampire biter) {
             int sucked = (int) Math.ceil((amt * perc));
             player.getFoodData().setFoodLevel(amt - sucked);
             player.causeFoodExhaustion(1000F);
-            if (!player.hasEffect(ModEffects.sanguinare) && Helper.canTurnPlayer(biter, player) && Helper.canBecomeVampire(player)) {
-                if (!player.isCreative()) SanguinareEffect.addRandom(player, true);
-            }
```

Comment:
@maxanier how are vampire player infected by mobs or should they not be able to do this? Without this there is no way they can do it. VampirePlayer#tryInfect is only called from the InfectAction

Reply:
Mhh, can't remember this, so it is probably not really intentional.
Maybe we can add an according entity action. For now I just added the according tryInfect hook to the vampire base entity attack